src/core/server/csp/index.ts

```diff
@@ -9,4 +9,5 @@
 import { CspConfig, ICspConfig } from './csp_config';
```

Comment:
__question:__ out of curiosity, is there any benefits in doing `import` + `export` comparing to only using `export`'s like this or is it just a matter of personal taste?

```ts
export type { ICspConfig } from './csp_config';
export type { CspConfigType } from './config';

export { config } from './config';
export { CspConfig } from './csp_config';
```

Reply:
> out of curiosity, is there any benefits in doing import + export comparing to only using export's like this or is it just a matter of personal taste?

I don't think it provides extra benefits. The compiler already knows how the imported code is used in the file. So it can remove all the imports used as interfaces or types.